### DISCUSSION TITLE
添加新的节点

### DISCUSSION THREAD

**Discussion by zhangyatao123456** [crate/crate#11664] [General]
添加写的节点无法数据重新平衡！我本来有4个节点，现在添加一个新的节点，查看监控，这个数据节点已经在集群里了，但是查看监控发现Started==0数据也没有改变，但是我创建写的表，数据可以分配到新的节点上来，为什么老的数据不能平衡呢？

> Added Google Translate:
> Data cannot be rebalanced for added write nodes! I originally had 4 nodes, now I add a new node, check the monitoring, the data node is already in the cluster, but check the monitoring and find that the Started==0 data has not changed, but I created the written table, the data can be distributed to New nodes come up, why can't the old data be balanced?

**Comment by jayeff** [comment]
Hi @zhangyatao123456,

normally CrateDB will automatically rebalance shards over all available nodes so without further information I don't know why this happens.

If you create a new sharded table are the shards for this table be distributed over all nodes (including new node)? As we discussed shard allocation filtering [earlier](https://github.com/crate/crate/issues/11636#issuecomment-896658544) is there potentially a filter set for this table?
